src/internal/modules/contact-entry/routes.js

```diff
@@ -0,0 +1,130 @@
+const preHandlers = require('./pre-handlers');
+const Joi = require('@hapi/joi');
+const controllers = require('./controllers');
+
+module.exports = {
+  getNew: {
```

Comment:
I think this is an unnecessary route and can be merged with getSelectAccountType.

Reply:
I agree completely that this is a useless route at the moment. 

I added it for future-proofing, so that in the future when we want to call the contact-entry module, we can pre-load it with a config object that states whether the selected object should be:
1. New, existing or either
2. A person, a company, or either

This would pre-populate the session object and then skip into the right parts of the flow. I hope this makes sense.